src/utils/neurons.utils.ts

```diff
@@ -0,0 +1,78 @@
+import type {
+  Ballot,
+  BallotInfo,
+  NeuronInfo,
+  ProposalId,
+  ProposalInfo,
+  Vote,
+} from "../types/governance_converters";
+
+const voteForProposal = ({
+  recentBallots,
+  proposalId,
+}: {
+  recentBallots: BallotInfo[];
+  proposalId: ProposalId | undefined;
+}): Vote | undefined => {
+  const ballot: BallotInfo | undefined = recentBallots.find(
+    ({ proposalId: id }: BallotInfo) =>
+      proposalId !== undefined && id === proposalId
+  );
+  return ballot?.vote;
+};
+
+export const ineligibleNeurons = ({
+  neurons,
+  proposal,
+}: {
+  neurons: NeuronInfo[];
+  proposal: ProposalInfo;
+}): NeuronInfo[] => {
+  const { ballots, proposalTimestampSeconds } = proposal;
+
+  return neurons.filter(({ createdTimestampSeconds, neuronId }: NeuronInfo) => {
+    const createdSinceProposal: boolean =
+      createdTimestampSeconds > proposalTimestampSeconds;
```

Comment:
Not sure but sounds like `>=`

Reply:
In Flutter they do following:

```
if (neuron.createdTimestampSeconds
                                  .secondsToDateTime()
                                  .isAfter(widget.proposal.proposalTimestamp)) {
                                  ```
                                  
"Returns true if this occurs after other." ([doc](https://api.flutter.dev/flutter/dart-core/DateTime/isAfter.html)). That's why I went with `>` but maybe I am missing something?